### PR TITLE
Cell Systems checklist

### PR DESCRIPTION
**Pull request by trangdata**
- add acknowledgment, funding, author contributions
- shorten abstract
- add highlights
- add key resources table

### CODE REVIEW

File: content/30.methods.md [hunk 1]
```diff
@@ -1,6 +1,19 @@
 ## STAR Methods
 ### Key resources table
-
+| **REAGENT or RESOURCE**          | **SOURCE**                                           | **IDENTIFIER**                                         |
+| -------------------------------- | ---------------------------------------------------- | ------------------------------------------------------ |
+| _Software and algorithms_        |
+| Data and code Zenodo deposit     | This paper                                           | http://dx.doi.org/10.5281/zenodo.5014756               |
```
Comment (dhimmel) [inline content/30.methods.md]:
```suggestion
| Data and code Zenodo deposit     | This paper                                           | https://doi.org/10.5281/zenodo.5014756               |
```

preferred doi URL format

File: content/30.methods.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+| Source code on GitHub            | This paper                                           | https://github.com/greenelab/iscb-diversity            |
+| Browsable Python and R notebooks | This paper                                           | https://greenelab.github.io/iscb-diversity/            |
+| PubmedPy                         | This paper                                           | https://github.com/dhimmel/pubmedpy                    |
+| Manubot                          | Himmelstein et al., 2019                             | http://dx.doi.org/10.1371/journal.pcbi.1007128         |
```
Comment (dhimmel) [inline content/30.methods.md]:
```suggestion
| Manubot                          | Himmelstein et al., 2019                             | https://doi.org/10.1371/journal.pcbi.1007128         |
```

File: content/01.abstract.md [hunk 3]
```diff
@@ -1,13 +1,20 @@
+## Highlights
+<!-- 3–4 bullet points of no more than 85 characters in length, including spaces,  -->
+- proportion of women honorees was similar to that of the field
+- scientists of East Asian origin have been underrepresented among honorees
+- disparities arise partly, but not exclusively, from geography
+- honorees with an affiliation in the US were overrepresented by a factor of 2.0
```
Comment (cgreene) [inline content/01.abstract.md:6]:
```suggestion
- honorees with an affiliation in the United States were overrepresented by a factor of 2.0
```
Not sure if they will require this 🤷

File: content/26.acknowledgments.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+We would like to thank the Gordon and Betty Moore Foundation whose support makes the study possible (GBMF4552 to D.S.H. and GBMF4552 to C.S.G.). 
+We thank the reviewers and editors for their insightful input to improve the quality of the study.
+
+## Author Contributions
+
+Conceptualization, C.S.G.;
+Methodology, T.T.L., D.S.H., A.A.H., M.R.G. and C.S.G.;
+Software, T.T.L., D.S.H., A.A.H. and M.R.G.;
+Investigation T.T.L., D.S.H., A.A.H., M.R.G. and C.S.G.;
+Data Curation, D.S.H., A.A.H. and M.R.G.;
```
Comment (cgreene) [inline content/26.acknowledgments.md]:
```suggestion
Data Curation, D.S.H., A.A.H., M.R.G. and C.S.G.;
```
The rare project where I curated things (Initial honoree table)!

**Review by cgreene — APPROVED** [review]
🚀

File: content/01.abstract.md (hunk 3 above)
Comment (trangdata) [reply content/01.abstract.md:6]: This is 89 characters, over the 85 limit...

File: content/26.acknowledgments.md (hunk 4 above)
Comment (trangdata) [reply content/26.acknowledgments.md]: Ah true. Sorry I forgot about that. I contributed to that, too! I'll add.

**Pull request merged**